### ISSUE TITLE
Sequelize v5, findById() was replaced by findByPk().

### ISSUE THREAD

**Issue by ACPK** [the-road-to-graphql/the-road-to-graphql#27]

**Comment by morenoh149** [comment]
yes it was. A PR updating the manuscript in this repo and the code in https://github.com/the-road-to-graphql/fullstack-apollo-express-postgresql-boilerplate would be very appreciated!

**Comment by rwieruch** [comment]
Thanks for pointing this out! And thanks @morenoh149 for jumping in 👍 

Any PR is more than welcome! Otherwise I keep this issue open for the next edition of the book.

Places to replace findById() with findByPk() by opening a PR:

- [ ] https://github.com/the-road-to-graphql/fullstack-apollo-express-postgresql-boilerplate
- [ ] https://github.com/the-road-to-graphql/the-road-to-graphql
- [ ] https://github.com/rwieruch/blog_robinwieruch_content/blob/master/graphql-apollo-server-tutorial.md

**Comment by Swap76** [comment]
Hey in  https://github.com/the-road-to-graphql/fullstack-apollo-express-postgresql-boilerplate this repo I am making PR for changing findById to findByPk.

This kind of my first PR if anything is there, please let me will improve on that
